packages/solid/tsup.config.ts

```diff
@@ -1,31 +1,13 @@
-import { defineConfig } from 'tsup';
-// eslint-disable-next-line import/no-namespace
-import * as preset from 'tsup-preset-solid';
-
-const presetOptions: preset.PresetOptions = {
-  entries: [
-    {
-      entry: 'src/index.ts',
-    },
-  ],
-  // Set to `true` to remove all `console.*` calls and `debugger` statements in prod builds
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  drop_console: false,
-  // Set to `true` to generate a CommonJS build alongside ESM
-  // cjs: true,
-};
-
-export default defineConfig((config) => {
-  const watching = Boolean(config.watch);
-
-  const parsedOptions = preset.parsePresetOptions(presetOptions, watching);
-
-  if (!watching) {
-    const packageFields = preset.generatePackageExports(parsedOptions);
-
-    // will update ./package.json with the correct export fields
-    preset.writePackageJson(packageFields);
-  }
-
-  return preset.generateTsupOptions(parsedOptions);
-});
+import { solidPlugin } from 'esbuild-plugin-solid';
+import { defineConfig, type Options } from 'tsup';
+
+export default defineConfig((options: Options) => ({
+  entry: ['./src/index.ts'],
+  clean: true,
+  dts: true,
+  format: ['esm'],
+  tsconfig: './tsconfig.build.json',
+  external: ['solid-js'],
+  esbuildPlugins: [solidPlugin()],
+  ...options,
```

Comment:
why do we need to spread the options object here ? apologies, it's been a while since I reviewed the PR, so I might be missing something

Reply:
resolved in https://github.com/LottieFiles/dotlottie-web/pull/373/commits/cc0633b0e59aa82cda17aedadb7d906884a5ba81